codemeta.json

```
@@ -0,0 +1,64 @@
+{
+    "@context": "https://w3id.org/codemeta/3.0",
+    "type": "SoftwareSourceCode",
+    "applicationCategory": "ecology",
+    "programmingLanguage": "julia",
+    "developmentStatus": "active",
+    "codeRepository": "https://github.com/EcoJulia/EcoBase.jl",
+    "name": "EcoBase.jl",
+    "issueTracker": "https://github.com/EcoJulia/EcoBase.jl/issues",
+    "readme": "https://github.com/EcoJulia/EcoBase.jl/blob/main/README.md",
+    "buildInstructions": "https://github.com/EcoJulia/EcoBase.jl/blob/main/README.md",
+    "dateCreated": "2018-01-30",
+    "operatingSystem": [
+        "Linux",
+        "Windows",
+        "macOS"
+    ],
+    "version": "v0.1.7",
+    "dateModified": "2024-07-21",
+    "datePublished": "2018-09-20",
+    "downloadUrl": "https://github.com/EcoJulia/EcoBase.jl/archive/refs/tags/v0.1.7.tar.gz",
+    "license": "https://spdx.org/licenses/MIT",
+    "author": [
+        {
+            "type": "Person",
+            "givenName": "Richard",
+            "familyName": "Reeve",
+            "email": "richard.reeve@glasgow.ac.uk",
+            "id": "https://orcid.org/0000-0003-2589-8091",
+            "affiliation": [
+                {
+                    "type": "Organization",
+                    "name": "University of Glasgow",
+                    "identifier": "https://ror.org/00vtgdb53"
+                }
+            ]
+        },
+        {
+            "type": "Person",
+            "givenName": "Michael",
+            "familyName": "Borregaard",
+            "id": "https://orcid.org/0000-0002-8146-8435",
+            "affiliation": [
+                {
+                    "type": "Organization",
+                    "name": "University of Copenhagen",
+                    "identifier": "https://ror.org/035b05819"
+                }
+            ]
+        }
```

```suggestion
        },
        {
            "type": "Person",
            "givenName": "Kevin",
            "familyName": "Bonham",
            "id": "https://orcid.org/0000-0003-3200-7533",
            "affiliation": [
                {
                    "type": "Organization",
                    "name": "Wellesley College",
                    "identifier": "https://ror.org/01srpnj69"
                }
            ]
        }
```

Only if you want to include me above (which again, I would not insist on)

Also, there's a non-trivial chance that my affiliation is going to change in the next few months, but I can update as necessary if that happens.

---

Thanks - I've added that in. But the only place you need to change anything in is the `ror` in `Project.toml`, and then rerun the crosswalk, which you can do by running `ResearchSoftwareMetadata.crosswalk()` and it will update everywhere else.